### PR TITLE
Update codeowners with input from Yuri (PR#51)

### PR DESCRIPTION
**Pull request by starsdong**
PR #51  merged into this request with the latest CODEOWNERS list

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,17 +1,20 @@
 *                                                  @plexoos           @nigmatkulov
 /OnlTools                                          @kehw              @plexoos          @jml985
 /StDb                                              @dmarkh            @plexoos
-/StDb/idl                                          @akioogawa @dmarkh         @fgeurts          @iraklic @plexoos        @yfisyak
+/StDb/idl                                          @akioogawa 	      @dmarkh		@fgeurts	@iraklic	@plexoos	@yfisyak
+/StRoot/GenFit                                     @yfisyak
+/StRoot/KFParticle*                                @yfisyak
```
Comment (plexoos) [inline .github/CODEOWNERS]: Our repository does not contain these subdirectories

Comment (starsdong) [reply .github/CODEOWNERS]: Thanks, Dmitri. I removed these two lines.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -38,12 +40,14 @@
 /StRoot/StDetectorDbMaker/St_tpcSCGLC*             @genevb
 /StRoot/StDetectorDbMaker/St_trigDetSums*          @genevb
 /StRoot/StDetectorDbMaker/St_vertexSeed*           @genevb
+/StRoot/StEandBDirMaker                            @yfisyak
```
Comment (plexoos) [inline .github/CODEOWNERS]: No such file or directory

```suggestion
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -53,22 +57,29 @@
 /StRoot/StFpd*                                     @akioogawa
 /StRoot/StFstSimMaker				   @jdbrice
 /StRoot/StFttSimMaker				   @jdbrice
-/StRoot/StGenericVertexMaker                       @plexoos           @perevbnlgov	@akioogawa	@klendathu2k
+/StRoot/StGenericVertexMaker                       @plexoos           @perevbnlgov	@akioogawa	@klendathu2k   @yfisyak
+/StRoot/StGmt*                                     @yfisyak
```
Comment (plexoos) [inline .github/CODEOWNERS]: No such file or directory

```suggestion
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -53,22 +57,29 @@
...
 /StRoot/StHeavyPool                                @starsdong         @ullrich-bnl 	@fgeurts
 /StRoot/StHighPtTagsMaker                          @ullrich-bnl
 /StRoot/StHltMaker				   @kehw
 /StRoot/StIOMaker				   @plexoos           @perevbnlgov
 /StRoot/StIst*					   @plexoos	      @starsdong
 /StRoot/StJetFinder                                @zlchang           @veprbl
 /StRoot/StJetMaker                                 @zlchang           @veprbl
+/StRoot/StKFParticleAnalysisMaker                  @yfisyak
+/StRoot/StKFVertexMaker                            @yfisyak
```
Comment (plexoos) [inline .github/CODEOWNERS]: No such file or directory

```suggestion
```

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -53,22 +57,29 @@
...
+/StRoot/StMagF                                     @yfisyak
+/StRoot/StMc*                                      @klendathu2k       @perevbnlgov
 /StRoot/StMcAnalysisMaker			   @klendathu2k
 /StRoot/StMcEvent*				   @klendathu2k
 /StRoot/StMiniDstMaker                             @ullrich-bnl
 /StRoot/StMtd*                                     @marrbnl
 /StRoot/StMuDSTMaker                               @jdbrice
 /StRoot/StMuDSTMaker/EMC                           @kkauder           @rkunnawa
 /StRoot/StMuDSTMaker/RICHTOF                       @fgeurts           @starsdong
+/StRoot/StMuMcAnalysisMaker                        @yfisyak
```
Comment (plexoos) [inline .github/CODEOWNERS]: No such file or directory

```suggestion
```

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -79,6 +90,10 @@
 /StRoot/StSpinPool                                 @akioogawa         @zlchang     @veprbl
 /StRoot/StSsd*					   @starsdong	      @plexoos
 /StRoot/StSst*					   @starsdong	      @plexoos
+/StRoot/StStarLogger                               @yfisyak
+/StRoot/StTMVARank                                 @yfisyak
```
Comment (plexoos) [inline .github/CODEOWNERS]: No such file or directory

```suggestion
```

File: .github/CODEOWNERS [hunk 7]
```diff
@@ -88,25 +103,33 @@
 /StRoot/StUtilities/StMess*                        @genevb
 /StRoot/StUtilities/StMultiH*                      @genevb
 /StRoot/StVpd*                                     @ZaochenYe         @fgeurts
+/StRoot/StXTrakMaker                               @perevbnlgov
+/StRoot/StxMaker                                   @yfisyak
```
Comment (plexoos) [inline .github/CODEOWNERS]: No such file or directory

```suggestion
```

File: .github/CODEOWNERS [hunk 8]
```diff
@@ -88,25 +103,33 @@
...
 /StRoot/St_QA_Maker                                @genevb
 /StRoot/St_TLA_Maker				   @perevbnlgov
+/StRoot/St_base                                    @perevbnlgov       @yfisyak
+/StRoot/St_baseTest                                @perevbnlgov       @yfisyak
```
Comment (plexoos) [inline .github/CODEOWNERS]:
```suggestion
/StRoot/St_base*                                    @perevbnlgov       @yfisyak
```

File: .github/CODEOWNERS [hunk 9]
```diff
@@ -88,25 +103,33 @@
...
 /StRoot/St_db_Maker                                @dmarkh            @plexoos     @perevbnlgov
-/StRoot/St_geant_Maker				   @klendathu2k	      @perevbnlgov
+/StRoot/St_geant_Maker				   @klendathu2k	      @perevbnlgov @yfisyak
+/StRoot/St_geom_Maker                              @perevbnlgov
```
Comment (plexoos) [inline .github/CODEOWNERS]: No such file or directory
```suggestion
```

File: .github/CODEOWNERS [hunk 10]
```diff
@@ -119,7 +142,20 @@
...
 /StarDb/emc					   @kkauder           @rkunnawa
 /StarVMC					   @klendathu2k
-/mgr                                               @genevb            @klendathu2k
-/pams                                              @klendathu2k
+/StarVMC/geant3                                    @yfisyak
+/StarVMC/Geometry                                  @klendathu2k
+/StarVMC/GeoTestMaker                              @perevbnlgov
+/StarVMC/minicern                                  @perevbnlgov
+/StarVMC/pgf77                                     @yfisyak
+/StarVMC/Rotations                                 @yfisyak
```
Comment (plexoos) [inline .github/CODEOWNERS]: No such file or directory
```suggestion
```

File: .github/CODEOWNERS [hunk 11]
```diff
@@ -119,7 +142,20 @@
...
+/StarVMC/StarAgmlLib                               @klendathu2k
+/StarVMC/StarAgmlUtil                              @klendathu2k
```
Comment (plexoos) [inline .github/CODEOWNERS]:
```suggestion
/StarVMC/StarAgml*                               @klendathu2k
```

File: .github/CODEOWNERS [hunk 12]
```diff
@@ -119,7 +142,20 @@
...
+/StarVMC/StarBASE                                  @klendathu2k
+/StarVMC/StarGeometry                              @klendathu2k
+/StarVMC/StarVMCApplication                        @yfisyak
+/StarVMC/StVMCMaker                                @yfisyak
```
Comment (plexoos) [inline .github/CODEOWNERS]: No such file or directory
```suggestion
```

**Review by plexoos — CHANGES_REQUESTED** [review]

**Comment by plexoos** [comment]
Ready to merge?

**Pull request merged**